Ladybird/EventLoopImplementationQt.cpp

```diff
@@ -149,16 +150,21 @@ void EventLoopManagerQt::unregister_notifier(Core::Notifier& notifier)
 
 void EventLoopManagerQt::did_post_event()
 {
-    m_process_core_events_timer.start();
+    QCoreApplication::postEvent(m_main_thread_event_target.ptr(), new QtEventLoopManagerEvent(QtEventLoopManagerEvent::process_event_queue_event_type()));
```

Comment:
Am I reading this right that we are allocating a new fancy QEvent for every Core::Event? Can we just create one event in the constructor and post it a billion times? Or is that not how QEvents work.

Reply:
Posting the event gives ownership of the allocated event to Qt, so it will delete it out from under us if we try to reuse it. I would like it if we found another way to do this, but I did try a few things before settling on this.